lib/entities/state/attributes/CarpetTurboStateAttribute.js

```diff
@@ -0,0 +1,32 @@
+const StateAttribute = require("./StateAttribute");
```

Comment:
Quoting from the previous PR:
> Since this is just a setting, it should not be a state attribute but rather a simple entity in `core/entities`


And to answer to your response:

>I know it's not ideal, I'm not sure if there's a better way to do it. 

Due to the magic of javascript, you can simply cache it in a dynamic property of the Viomi Vacuum implementation.
e.g. `this.carpetModeState = new ValetudoCarpetModeState(....)` boom it's there.

I don't want it in the state, because it isn't state. It's just a configuration

>I'm also not sure what you have in mind with the entities.

Entities should be used for things that aren't primitives and also aren't part of the state of the robot.
e.g. Settings such as the `ValetudoDNDConfiguration` which is just a bunch of primitives that require each other to make any sense wrapped into a single thing




And you can scratch some of the above since carpet mode is just a boolean. I'll just keep it here for reference.

Anyways, since carpet mode will most likely stay a single boolean, we can ditch this wrapper altogether.